rules/choosing-authentication/rule.md

```diff
@@ -140,28 +127,58 @@ Advantages:
 * Allows you to define custom authentication logic
 * Supports multiple applications/identity consumers
 * Supports multiple clients and client types
-* Supports OAuth/OIDC
+* Fully OIDC compliant
 
 ::: bad
 Disadvantages:
 :::
 
-* Steep learning curve
-* Requires additional setup
-* Requires additional skillset
-* Requires additional management
+* Learning curve
+* Requires additional hosting resources
+* Requires additional skill set
+* Requires ongoing maintenance
```

Comment:
Is it worth mentioning the cost here (as some of the other options are free - at least at low usage)?

Reply:
Good point - added